Fix EcoDim ED-10011

bartrikers6224 opened the pull request:
The EcoDim ED-10011 actually sends the actions in the payload including the postfix "_1".

![image](https://user-images.githubusercontent.com/83872170/117713125-899f4900-b1d5-11eb-94e5-3d8050734ed1.png)

Koenkk commented:
@bartrikers6224 I don't know how the domitcz integration works, but if both have the same endpoint I propose to revert this change and remove `multiEndpoint: true` from this device.

bartrikers6224 commented:
Ok, I've reverted this change and removed multiEndpoint:true and created a new [pull-request](https://github.com/Koenkk/zigbee-herdsman-converters/pull/2582)
